EmotiBit.cpp

```diff
@@ -2110,12 +2082,9 @@ bool EmotiBit::processThermopileData()
 	uint32_t timestampAmb;
 	uint32_t timestampSto;
 
-	static const unsigned long int samplingInterval = 1000000 / (_samplingRates.thermopile * _samplesAveraged.thermopile);
-#ifdef ADAFRUIT_FEATHER_M0
-	static const unsigned int minSwapTime = max(500, min(samplingInterval / 10, 10000));
-#elif defined ARDUINO_FEATHER_ESP32
-	static const unsigned int minSwapTime = _max(500, _min(samplingInterval / 10, 10000));
-#endif
+	static const int samplingInterval = 1000000 / (_samplingRates.thermopile * _samplesAveraged.thermopile);
```

Comment:
Should this be `samplingInterval = (1000000 / _samplingRates.thermopile) * _samplesAveraged.thermopile;`

Otherwise, `samplingInterval` has the dimension sample^2/sec